src/test/java/hw5/pages/UserTablePage.java

```diff
@@ -0,0 +1,95 @@
+package hw5.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class UserTablePage {
+
+    protected WebDriver driver;
```

Comment:
May be it will be better extract this field to the base class?

Reply:
Done. Moved to BasePage class for 3 pages.